keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ClusterDeployStrategy.kt

```diff
@@ -36,7 +14,6 @@ sealed class ClusterDeployStrategy {
   }
 }
 
-@JsonTypeName("red-black")
 data class RedBlack(
```

Comment:
To add to the point above on the `sealed` base class, I feel like these should not be in `keel-api` as they are resource-specific.

Reply:
I had the same thought but is that actually true? The existing strategies (red black and highlander) are really just abstractions of the steps taken around the actual deployment itself. Those steps are equally applicable to any cloud provider that has a concept of a cluster.

Reply:
Hmm, when you put it that way... Fair enough. 🙂